packages/kbn-server-route-repository/src/register_routes.ts

```diff
@@ -55,12 +57,7 @@ export function registerRoutes<TDependencies extends Record<string, any>>({
       response: KibanaResponseFactory
     ) => {
       try {
-        const runtimeType = params || t.strict({});
-
-        const validatedParams = decodeRequestParams(
-          pick(request, 'params', 'body', 'query'),
-          runtimeType
-        );
+        const validatedParams = validateParams(request, params);
```

Comment:
is this still throwing if there is no type defined, but request parameters were set?

Reply:
No, will fix!